go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/pulumi/pulumi-yaml
 
-go 1.21
+go 1.22
+
+toolchain go1.23.5
```

Comment:
I think this is making the test fail.  I don't think we need the `toolchain` line here?